ulib/experimental/Steel.Heap.fsti

```diff
@@ -46,13 +46,19 @@ val core_ref : Type u#0
     and a [pcm] governing it, for ease of type inference *)
 let ref (a:Type u#a) (pcm:pcm a) : Type u#0 = core_ref
 
+val core_ref_null : core_ref
+
 (** [null] is a specific reference, that is not associated to any value
 *)
-val null (#a:Type u#a) (#pcm:pcm a) : ref a pcm
+let null (#a:Type u#a) (#pcm:pcm a) : ref a pcm = core_ref_null
+
+(** Checking whether [r] is the null pointer is decidable through [is_null]
+*)
+val core_ref_is_null (r:core_ref) : b:bool { b <==> r == core_ref_null }
 
 (** Checking whether [r] is the null pointer is decidable through [is_null]
 *)
-val is_null (#a:Type u#a) (#pcm:pcm a) (r:ref a pcm) : (b:bool{b <==> r == null})
+let is_null (#a:Type u#a) (#pcm:pcm a) (r:ref a pcm) : (b:bool{b <==> r == null}) = core_ref_is_null r
```

Comment:
This makes it evident that is_null does not depend on `a` or the `pcm`, useful for further positivity checks in modules that depend on Heap.ref